### PR TITLE
Add ko

### PR DESCRIPTION
**Pull request by kagg-design**

### CODE REVIEW

**Comment by kagg-design** [comment]
I have fixed .pot and .po files, but it is not needed, actually.

File: .tests/php/integration/AMainPluginFileTest.php [hunk 1]
```diff
@@ -30,7 +34,7 @@ public function tearDown(): void {
 	 * Test main plugin file content.
 	 */
 	public function test_main_file_content(): void {
-		self::assertSame( '1.9.0', HCAPTCHA_VERSION );
+		self::assertSame( '1.9.1', HCAPTCHA_VERSION );
```
Comment (e271828-) [inline .tests/php/integration/AMainPluginFileTest.php:37]: Does it make sense to hard-code this into a test, vs. e.g. just checking it's semVer?

Comment (kagg-design) [reply .tests/php/integration/AMainPluginFileTest.php:37]: I prefer to check everything, including the version number. It is not a big issue to fix the test.

**Comment by kagg-design** [comment]
I have imported Korean translation to GlotPress: https://translate.wordpress.org/projects/wp-plugins/hcaptcha-for-forms-and-more/stable/ko/default/. We have to wait, the Korean editor will accept/fix it soon.

**Comment by e271828-** [comment]
> I have imported Korean translation to GlotPress: https://translate.wordpress.org/projects/wp-plugins/hcaptcha-for-forms-and-more/stable/ko/default/. We have to wait, the Korean editor will accept/fix it soon.

But there is no reason to block a release, correct? It will simply become available in the current release when approved?

**Comment by kagg-design** [comment]
No reason to delay the release. Translation will be approved in some days and will become available immediately to everybody using ko_KR locale.

**Comment by kagg-design** [comment]
Please wait for a moment... I forgot to set the proper filename for po/mo! Doing a commit...

**Comment by kagg-design** [comment]
Fixed filenames. Please restart the action.

**Pull request merged**